CafeRecords/Pages/Index.cshtml.cs

```diff
@@ -24,13 +20,13 @@ public void OnGet()
         {
             List<Owner> cafeOwners = new List<Owner>();
 
-            string CafeJsonString = GetData("https://data.cityofchicago.org/resource/mqmh-p6ud.json");
-            Cafe[] allCafes = Cafe.FromJson(CafeJsonString);
-            ViewData["allCafes"] = allCafes;
+            string cafeJsonString = GetData("https://data.cityofchicago.org/resource/mqmh-p6ud.json");
+            Cafe[] allCafes = Cafe.FromJson(cafeJsonString);
```

Comment:
Changed the notation of the variable to Camel Case notation.

Reply:
Yes, this looks better.